src/libs/LocaleDigitUtils.js

```diff
@@ -0,0 +1,73 @@
+import _ from 'underscore';
+
+import * as NumberFormatUtils from './NumberFormatUtils';
+
+const STANDARD_DIGITS = Object.freeze(['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '.', '-', ',']);
```

Comment:
Do we have any advantage of using `Object.freeze` here? Trying to understand.

Reply:
I intended to export `getLocaleDigits` function at beginning. `Object.freeze` prevents the returned result from being accidentally changed by external code. Due to it's an implement detail now I think it can be safely removed.